Assets/Plugins/iOS/Shopify/MessageCenter.swift

```diff
@@ -32,14 +32,14 @@ import PassKit
     private static var messages = [String: UnityMessage]()
     
     static func send(_ message: UnityMessage) {
-        UnityInterfaceWrapper.sendMessage(try! message.serializedString(), toObject: message.recipientObjectName, havingMethodName: message.recipientMethodName)
+        MessageSender.sendMessage(try! message.serializedString(), object: message.object, method: message.method)
     }
     
     static func send(_ message: UnityMessage, completionHandler: UnityMessage.MessageCompletion?) {
         
         messages[message.identifier] = message
         
-        UnityInterfaceWrapper.sendMessage(try! message.serializedString(), toObject: message.recipientObjectName, havingMethodName: message.recipientMethodName)
+        MessageSender.sendMessage(try! message.serializedString(), object: message.object, method: message.method)
```

Comment:
Actually I realized we don't need this class at all, we can define the C++ method in the bridging header